Sources/SwiftInspectorVisitors/GenericRequirementVisitor.swift

```diff
@@ -31,14 +31,20 @@ public final class GenericRequirementVisitor: SyntaxVisitor {
 
   public override func visit(_ node: SameTypeRequirementSyntax) -> SyntaxVisitorContinueKind {
     genericRequirements.append(GenericRequirement(node: node))
-    return .visitChildren
+    // We only care about siblings, not children.
```

Comment:
Let's say _why_ here, not what. This is quite a cat comment atm 😁 

![](https://preview.redd.it/iuy9fxt300811.png?auto=webp&s=156f5ffebbea5b30189eb36012e5bc7e3369c381)

Reply:
1caf37d99c1cbd555b1f4e26b08de4eaf18fc800